docs/diagrams/SortSequenceDiagram.puml

```diff
@@ -0,0 +1,94 @@
+@startuml
+
+!include style.puml
+skinparam ArrowFontStyle plain
+
+box Logic LOGIC_COLOR_T1
+participant ":LogicManager" as LogicManager LOGIC_COLOR
+participant ":StaffConnectParser" as StaffConnectParser LOGIC_COLOR
+participant ":SortCommandParser" as SortCommandParser LOGIC_COLOR
+participant "<<Class>>\nNameComparator" as NameComparator LOGIC_COLOR
+participant "c\n:NameComparator" as NAME_COMPARATOR LOGIC_COLOR
```

Comment:
nit: At line 10, `<<Class>>` -> `<<class>>`
nit: At line 11, should the "\n" be removed within the object name?

Reply:
Amended